src/RequestProcessor.cs

```diff
@@ -98,6 +103,16 @@ internal StreamingMessage ProcessFunctionLoadRequest(StreamingMessage request)
             {
                 // Try loading the metadata of the function
                 _functionLoader.Load(functionLoadRequest);
+
+                // if we haven't yet, add the well-known Function App module path to the PSModulePath
+                // The location of this module path is in a folder called "Modules" in the root of the Function App.
+                if (!_prependedPath)
+                {
+                    string functionAppModulesPath = Path.GetFullPath(
+                        Path.Combine(functionLoadRequest.Metadata.Directory, "..", "Modules"));
+                    _powerShellManager.PrependToPSModulePath(functionAppModulesPath);
```

Comment:
As long as one worker handles only one Function App, doing this here is fine.
@pragnagopa is it guaranteed that one worker only handles one Function App?

Reply:
Checked with @pragnagopa and yes, one worker process only handles one Function App, for isolation purpose. So we are good here.

Reply:
Makes sense! That's the assumption I was working with since you typically do the `func start` in a Function App.

Good to know for sure 👍